apps/api/src/app/inbox/inbox.controller.ts

```diff
@@ -313,10 +316,26 @@ export class InboxController {
   }
 
   @UseGuards(AuthGuard('subscriberJwt'))
-  @Patch('/preferences/:workflowId')
+  @Patch('/preferences/bulk')
```

Comment:
the order matters, this one should be before the `PATCH /preferences/:workflowIdOrIdentifier`, otherwise the latter will handle the call

Reply:
Let's maybe add an in-code comment for this?